app.js

```diff
@@ -136,13 +143,12 @@ function dotabuff(bot, message) {
         var user = message.match[3];
         if (user in steamUserMapping) {
             request({
-                url: 'https://api.steampowered.com/IDOTA2Match_570/GetMatchHistory/V001/?key=' +
-                    process.env.STEAM_API_KEY + '&account_id=' + steamUserMapping[user],
+                url: 'https://api.opendota.com/api/players/' + steamUserMapping[user] + '/matches?limit=1',
                 json: true
             }, function (error, response, body) {
                 if (!error && response.statusCode === 200) {
-                    bot.reply(message, baseURL +
-                        body.result.matches[0].match_id);
+                    bot.reply(message, user + ' went ' + body[0].kills + '/' + body[0].deaths + '/' + body[0].assists +
```

Comment:
Is this the only info we want to grab? Couldn't we add flags to choose what we get?

Reply:
I do think we could add more information such as whether the match was a win or a loss, but it would take me a bit more effort because I would have to figure out if the user was on radiant or dire by checking the most significant bit of the 8-bit `player_slot` field. For the current commit I just used the subset of fields returned by the opendota matches API which could be inserted into the bot's response string with the least amount of work.